src/LockManager.php

```diff
@@ -209,6 +221,10 @@ public function unlock(Lock $lock)
 			$this->debug('> sent unlock request to '. $this->servers[$i][0] .':'. $this->servers[$i][1]);
 		}
 		
+		if (count($this->sockets) < $this->quorum) {
```

Comment:
It doesn't matter, if the client is connected to less than quorum servers then the lock has already been released.

If the servers are still active but the client connection has dropped, then the servers will have released the lock.

If the servers have died, then there won't be enough servers holding the lock for it to be valid.

Adding these Exceptions just means that the caller needs to trap errors in the unlock call that they don't care about.